.github/workflows/cd-protonet.yml

```diff
@@ -1,41 +1,26 @@
 name: Continuous Deployment (Protonet)
-## run after every successful CI job of new commits to the master branch
-#on:
-#  workflow_run:
-#    workflows: [Continuous Integration (Kava Master)]
-#    types:
-#      - completed
 
-jobs:
-  # in order:
-  # enter standby (prevents autoscaling group from killing node during deploy)
-  # stop kava
-  # take ebs + zfs snapshots
-  # download updated binary and genesis
-  # reset application database state (only done on internal testnet)
-  reset-chain-to-zero-state:
-    # only start cd pipeline if last ci run was successful
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
-    uses: ./.github/workflows/cd-reset-internal-testnet.yml
-    with:
-      aws-region: us-east-1
-      chain-id: proto_2221-17000
-      ssm-document-name: kava-testnet-internal-node-update
-      playbook-name: reset-protonet-playbook.yml
-      playbook-infrastructure-branch: master
-    secrets: inherit
+## run after every successful CI job of new commits to the master branch, or on manual dispatch
+on:
+  workflow_run:
+    workflows: [Continuous Integration (Kava Master)]
+    types:
+      - completed
+  workflow_dispatch:
```

Comment:
this workflow supports being triggered by either:
1. a successful run of the CI on merge to master
2. manual dispatch via the github UI

configuring it this way allows us to have a single file instead of two copies with different `on` conditions